arch/arm/boot/dts/imx6ul-ts7250v3.dtsi

```diff
@@ -0,0 +1,1030 @@
+			interrupts = <0>;
+		};
+
+		/* COM2 RS-232 */
+		fpga_uart1: serial@10 {
+			compatible = "ns16550a";
+			device_type = "serial";
+			reg = <0x10 16>;
+
+			current-speed = <115200>;
+			reg-shift = <1>;
+			reg-io-width = <1>;
+			fifo-size = <64>;
+			clock-frequency = <1843200>;
+			interrupt-parent = <&fpga_intc>;
+			interrupts = <1>;
+		};
+
+		/* COM3 RS-232 */
+		fpga_uart2: serial@20 {
+			compatible = "ns16550a";
+			device_type = "serial";
+			reg = <0x20 16>;
+
+			current-speed = <115200>;
+			reg-shift = <1>;
+			reg-io-width = <1>;
+			fifo-size = <64>;
+			clock-frequency = <1843200>;
+			interrupt-parent = <&fpga_intc>;
+			interrupts = <2>;
+		};
+
+		/* COM2 RS-485 */
+		fpga_uart3: serial@30 {
+			compatible = "ns16550a";
+			device_type = "serial";
+			reg = <0x30 16>;
+
+			current-speed = <115200>;
+			reg-shift = <1>;
+			reg-io-width = <1>;
+			fifo-size = <64>;
+			clock-frequency = <1843200>;
+			interrupt-parent = <&fpga_intc>;
+			interrupts = <3>;
+		};
+
+		/* COM2 RS-422 */
+		fpga_uart4: serial@40 {
+			compatible = "ns16550a";
+			device_type = "serial";
+			reg = <0x40 16>;
+
+			current-speed = <115200>;
+			reg-shift = <1>;
+			reg-io-width = <1>;
+			fifo-size = <64>;
+			clock-frequency = <1843200>;
+			interrupt-parent = <&fpga_intc>;
+			interrupts = <4>;
+		};
+
+		/* Mikrobus UART */
+		fpga_uart5: serial@50 {
+			compatible = "ns16550a";
+			device_type = "serial";
+			reg = <0x50 16>;
+
+			current-speed = <115200>;
+			reg-shift = <1>;
+			reg-io-width = <1>;
+			fifo-size = <64>;
+			clock-frequency = <1843200>;
+			interrupt-parent = <&fpga_intc>;
+			interrupts = <5>;
+		};
+
+		fpga_uart6: serial@60 {
+			compatible = "ns16550a";
+			device_type = "serial";
+			reg = <0x60 16>;
+
+			current-speed = <115200>;
+			reg-shift = <1>;
+			reg-io-width = <1>;
+			fifo-size = <64>;
+			clock-frequency = <1843200>;
+			interrupt-parent = <&fpga_intc>;
+			interrupts = <6>;
+		};
+
+		fpga_uart7: serial@70 {
+			compatible = "ns16550a";
+			device_type = "serial";
+			reg = <0x70 16>;
+
+			current-speed = <115200>;
+			reg-shift = <1>;
+			reg-io-width = <1>;
+			fifo-size = <64>;
+			clock-frequency = <1843200>;
+			interrupt-parent = <&fpga_intc>;
+			interrupts = <7>;
+		};
+
+		fpga_uart8: serial@80 {
+			compatible = "ns16550a";
+			device_type = "serial";
+			reg = <0x80 16>;
+
+			current-speed = <115200>;
+			reg-shift = <1>;
+			reg-io-width = <1>;
+			fifo-size = <64>;
+			clock-frequency = <1843200>;
+			interrupt-parent = <&fpga_intc>;
+			interrupts = <8>;
+		};
+
+		opencores_spi0: spi@100 {
+			compatible = "opencores,spi-oc";
+			#address-cells = <1>;
+			#size-cells = <0>;
+			reg = <0x100 32>;
+			interrupt-parent = <&fpga_intc>;
+			interrupts = <9>;
+			clocks = <&fpga_clk_weim_bclk>;
+			clock-names = "spi-oc-clk";
+			opencores-spi,idx = <0>;
+			opencores-spi,num-chipselects = <3>;
+
+			dioheader: spi@0 {
+				compatible = "technologic,spidev";
```

Comment:
I don't think we added an exception to spidev.c, in 5.10 this can safely be left as "spidev" it will give a warning but the bus will still function.